README.md

```diff
@@ -205,3 +168,39 @@ Endpoint | Action
 `/login` | Should create a cookie using `jwt.sign`, attach it to the response, and redirect to `/`
 `/logout` | Should remove the cookie and redirect to `/`
 `/auth_check` | Should check if the cookie exists, validate it with `jwt.verify`, and send back a 200 or 401 response with an informative message!
+
+## optional extra exercise 2- implement a HMAC
+
+Now we are going to make an program for handling and verifying important communications.
+
+Node.js has a built-in [HMAC function](https://nodejs.org/dist/latest-v8.x/docs/api/crypto.html#crypto_class_hmac).
+
+You will be provided a Node.js module (to be used in your terminal), that accepts a 'secret' and returns an object with two functions on it, which you have to implement:
+- sign: This function accepts a value (`String`), and uses the Node.js crypto module to create and return a HMAC `String` of that value using `SHA256` algorithm and `hex` encoder.
```

Comment:
minor change, I recommend `String` be changed to `string`. Uppercase references the js String global object.

Reply:
Marking as resolved now that the changes have been made.